spec/MaskedField_spec.jsx

```diff
@@ -874,21 +881,20 @@ describe('MaskedField', function() {
   });
 
   afterEach(function() {
-    ReactDOM.unmountComponentAtNode(container);
+    component.detach();
```

Comment:
I actually haven't unmounted a component after a test before. Just curious, what purpose does this have?

Reply:
We don’t normally have to do it, but these tests actually mount the components to the document (so they have focus and selections). That’s why I’m detaching them here.